packages/app/src/app/overmind/namespaces/dashboard/types.ts

```diff
@@ -10,6 +10,7 @@ export type PageTypes =
   | 'synced-sandboxes'
   | 'my-contributions'
   | 'repositories'
+  | 'repository-branches'
```

Comment:
this is not used as far as I can see. was the intention to have it as a separate page with a dedicated route? I see this is already working in the `/dashboard/repositories` page.

Reply:
@alexnm yep! It's a leftover... I started implementing a separate page but since we always fetch the branches to show the count, I realized we didn't need a separate page and went about implementing the same pattern we have for synced sandboxes.